tests/conftest.py

```diff
@@ -18,13 +18,18 @@
 def pytest_addoption(parser):
     """
     Additional command-line arguments passed to pytest.
-        --cpu: use CPU during testing (DEFAULT: GPU)
     """
     parser.addoption(
         "--cpu", action="store_true", help="pass that argument to use CPU during testing (DEFAULT: False = GPU)"
     )
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "run_only_on(device): runs the test only on a given device [CPU | GPU]",
+    )
+
+
```

Comment:
That's a nice trick to run `@pytest.mark.run_only_on("GPU")` by default while skipping `@pytest.mark.run_only_on("CPU")` by default.

I was thinking that `-m 'run_only_on("GPU") and not run_only_on("CPU")'` filters the same set of tests, but is more clunky than just `pytest ...`.

Cool!